app/recipe/tests/test_recipe_api.py

```diff
@@ -0,0 +1,166 @@
+import uuid
+
+from django.test import TestCase
+from django.urls import reverse
+
+from rest_framework import status
+from rest_framework.test import APIClient
+from core.models import Recipe, Ingredient
+
+from recipe.serializers import RecipeSerializer
+
+RECIPES_URL = reverse('recipe:recipe-list')
+
+
+def create_recipe(ingredients_count=5):
+    """
+    Creates a dummy recipe for testing
+    """
+    payload = build_random_recipe_payload(ingredients_count)
+
+    defaults = {
+        'name': payload['name'],
+        'description': payload['description'],
+    }
+
+    recipe = Recipe.objects.create(**defaults)
+
+    for ingredient in payload['ingredients']:
+        Ingredient.objects.create(recipe=recipe, **ingredient)
+
+    return recipe
```

Comment:
It's a little bit out of the scope of this project but take a look at how you can create factories using this tool: [FactoryBoy](https://factoryboy.readthedocs.io/en/latest/orms.html#the-djangomodelfactory-subclass)

Reply:
Pending to take a look at FactoryBoy